src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -31,6 +33,13 @@ export class RundownTimelineService implements RundownService {
   ) {}
 
   public async activateRundown(rundownId: string): Promise<void> {
+
+    const basicRundowns: BasicRundown[] = await this.rundownRepository.getBasicRundowns();
+    const activeRundown: BasicRundown | undefined = basicRundowns? basicRundowns.find(rundown => rundown.isActive()) : undefined;
+
+    if (activeRundown) {
+      throw new AlreadyActivatedException(`Unable to activate rundown, because: ${activeRundown.name} is active. `)
```

Comment:
The colon in the error message is unnecessary. A suggestion would be to also mention what `activeRundown.name` is. E.g. `Unable to activate rundown, because the rundown ${activeRundown.name} is active.`